cyber/tools/cyber_monitor/screen.cc

```diff
@@ -164,15 +164,15 @@ void Screen::HighlightLine(int lineNo) {
   if (IsInit() && lineNo < Height()) {
     SetCurrentColor(WHITE_BLACK);
     for (int x = 0; x < Width(); ++x) {
-      int ch = mvinch(lineNo + highlight_direction_, x);
+      chtype = mvinch(lineNo + highlight_direction_, x);
```

Comment:
no variable name?

Reply:
sorry，missed during copy-paste，will add it later .